declarations/index.d.ts

```diff
@@ -1,3 +1,5 @@
+export type { ResolveData } from "../lib/NormalModuleFactory";
```

Comment:
I think better to put this here https://github.com/webpack/webpack/blob/main/lib/index.js#L59, like we do for all public types

Reply:
Fixed